ui/components/app/account-menu/keyring-label.js

```diff
@@ -0,0 +1,37 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+import { useI18nContext } from '../../../hooks/useI18nContext';
+import { KEYRING_TYPES } from '../../../../shared/constants/hardware-wallets';
+
+export default function KeyRingLabel(props) {
+  const t = useI18nContext();
+
+  const { keyring } = props;
```

Comment:
[REQ] Could we destructure this in function signature?

Reply:
In our newer components we've been destructuring in the signature.  I think for now we continue that pattern, and if you have strong feelings about it, we can address it in an engineering meeting.